R/pkgdown.R

```diff
@@ -40,6 +42,23 @@ local_pkgdown_site <- function(path = NULL, meta = NULL, env = parent.frame()) {
     desc$write(file = path(path, "DESCRIPTION"))
     
     file_create(path(path, "_pkgdown.yml"))
+  } 
+
+  if (clone) {
+    if (is.null(path)) {
+      cli::cli_abort("Can only clone when {.arg path} is set.")
```

Comment:
why an else after an error?

Reply:
I think this `if` makes the logic most clear, but it doesn't matter.